content/2022-01-04-brigade-2.2.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Brigade 2.2.0"
+slug: "2022-01-04-brigade-2.2.0"
+description: "Brigade 2.2.0 has been released"
+date: "2022-01-04"
+author: "Kent Rancourt"
+summary: "Brigade 2.2.0 has been released."
+---
+
+Happy New Year, Brigadiers!
+
+We're starting the new year off right with our second minor release of Brigade 2.
+
+v2.2.0 continues the trend started with v2.1.0 by delivering just a handful of small, incremental improvements. Notably, in this release, we've corrected a bug in the user list endpoint and made it easy for script authors to designate jobs as fallible -- that is, _allowed to fail without forcing the overall workflow defined by the script to fail_. This is a great feature for cases where script authors may wish to treat a certain job's failure as informational in nature rather than fatal.
+
+To demonstrate, fallible jobs, consider this trivial fragment of a `brigade.ts` file:
```

Comment:
I'd suggest avoiding words like "trivial" and "easy" in blog posts, much like one tries to avoid them in docs. The reason is that if someone doesn't find it trivial/easy, that can be alienating and make them bounce off the project.

Good replacements could be "low-friction" or "minimal-effort".

Reply:
Good thought! Would "small fragment" suffice?

Reply:
Tentatively addressed in ce2e4c95445397fbde75c8e41c860716a8566cf3

Willing to change further if you think it needs it.

Reply:
> made it easy for script authors to designate jobs as fallible

I'd go with something like:
made it approachable for script authors to designate jobs as fallible

or more clearly:
added an option for script authors to designate jobs as fallible

Reply:
> added an option for script authors to designate jobs as fallible

☝️  I like this.

Reply:
Fixed in 730c328600d5b536096bd5520998fe059c19b787